service/view/duplicates.py

```diff
@@ -16,6 +17,15 @@
     }
 }
 
+doi_query = {
```

Comment:
@Steven-Eardley - I hope you don't mind, but I've simplified this query so that it just uses the query string search.  This saves us having to get the search interface to construct a hidden (and otherwise unused) facet over the identifier field, which has better performance.

Let me know what you think, and if that's ok, I'll merge shortly.